src/main/scala/de/frosner/dds/core/DDS.scala

```diff
@@ -642,4 +648,19 @@ object DDS {
     summarizeGroups(toBeGroupedValues.groupByKey())
   }
 
+  private[core] def lg2IntCeil(count:Long) : Integer = {
```

Comment:
Unless you have other functions in mind to add to this class, I don't think it's worth the trouble yet.